CONTRIBUTING.md

```diff
@@ -1,61 +1,130 @@
+  * [Pull Request Checklist](#pull-request-checklist)
+  * [Documentation Style](#documentation-style)
 
-## Reporting issues
+Welcome! We are glad that you want to contribute to our project! 💖
 
-If you believe you've discovered a bug in aya, please check if the bug is
-already known or [create an issue](https://github.com/aya-rs/aya/issues) on
-github. Please also report an issue if you find documentation that you think is
-confusing or could be improved.
+As you get started, you are in the best position to give us feedback on areas of
+our project that we need help with including:
 
-When creating a new issue, make sure to include as many details as possible to
-help us understand the problem. When reporting a bug, always specify which
-version of aya you're using and which version of the linux kernel.
+* Problems found during setting up a new developer environment
+* Gaps in our Quickstart Guide or documentation
+* Bugs in our automation scripts
 
-## Documentation
+If anything doesn't make sense, or doesn't work when you run it, please open a
+bug report and let us know!
 
-If you find an API that is not documented, unclear or missing examples, please
-file an issue. If you make changes to the documentation, please read
-https://doc.rust-lang.org/rustdoc/how-to-write-documentation.html and make sure
-your changes conform to the format outlined here
-https://doc.rust-lang.org/rustdoc/how-to-write-documentation.html#documenting-components.
+## Ways to Contribute
 
-If you want to make changes to the Aya Book, see the readme in the book repo
-https://github.com/aya-rs/book.
+We welcome many different types of contributions including:
 
-## Fixing bugs and implementing new features
+* New features
+* Builds, CI/CD
+* Bug fixes
+* Documentation
+* Issue Triage
+* Answering questions on Discord
+* Web design
+* Communications / Social Media / Blog Posts
+* Release management
 
-Make sure that your work is tracked by an issue or a (draft) pull request, this
-helps us avoid duplicating work. If your work includes publicly visible changes,
-make sure those are properly documented as explained in the section above.
+Not everything happens through a GitHub pull request. Please come to our
+[Discord](https://discord.gg/xHW2cb2N6G) and let's discuss how we can work
+together.
 
-### Running tests
-Run the unit tests with `cargo test`. See [Aya Integration Tests](https://github.com/aya-rs/aya/blob/main/test/README.md) regarding running the integration tests.
+## Find an Issue
 
-### Commits
+We have good first issues for new contributors and help wanted issues suitable
+for any contributor. [good first issue](https://github.com/aya-rs/aya/labels/good%20first%20issue) has extra information to
+help you make your first contribution. [help wanted](https://github.com/aya-rs/aya/labels/help%20wanted) are issues
+suitable for someone who isn't a core maintainer and is good to move onto after
+your first pull request.
```

Comment:
And here some sentences start without capital letters. :stuck_out_tongue: Anyway, I would still rephrase it entirely.

```suggestion
Issues labelled as ["good first issue"](https://github.com/aya-rs/aya/labels/good%20first%20issue)
are suitable for new contributors. They provide extra information to help you make your first
contribution.

Issues labelled as ["help wanted"](https://github.com/aya-rs/aya/labels/help%20wanted) are
usually more difficult. We recommend them for people who aren't core maintainers, but either
made some contributions already or feel comfortable with starting from more demanding tasks.
```